Amazon.SQS/src/Extensions.SQS/Bootstrapper/SqsExtensionProvider.cs

```diff
@@ -1,4 +1,6 @@
 ﻿
+using Microsoft.Azure.WebJobs;
```

Comment:
could you keep the using statements together? It's just a coding standard I like to follow?